src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -12,61 +19,65 @@ const underlinedItem = (color: string): ICSSInJSStyle => ({
 })
 
 const getActionStyles = ({
-  props: { primary, underlined, iconOnly, isFromKeyboard },
+  props: { underlined, iconOnly, pointing, vertical },
   variables: v,
-  color,
+  colors,
 }: {
   props: MenuItemPropsAndState
   variables: MenuVariables
-  color: string
-}): ICSSInJSStyle =>
-  underlined || iconOnly
-    ? {
-        color,
-      }
-    : primary
-    ? {
-        color: v.primaryActiveColor,
-        background: v.primaryActiveBackgroundColor,
-      }
-    : {
-        color,
-        background: v.activeBackgroundColor,
-      }
+  colors: ColorScheme
+}): ICSSInJSStyle => {
+  if (underlined || iconOnly) {
+    return { color: v.color }
+  }
+
+  if (pointing && vertical) {
+    return { background: v.activeBackgroundColor }
+  }
+
+  return {
+    color: colors.foreground.active,
+    background: colors.background.active,
+  }
+}
 
 const getFocusedStyles = ({
   props,
   variables: v,
-  color,
+  colors,
 }: {
   props: MenuItemPropsAndState
   variables: MenuVariables
-  color: string
+  colors: ColorScheme
 }): ICSSInJSStyle => {
-  const { primary, underlined, isFromKeyboard, active } = props
+  const { iconOnly, underlined, isFromKeyboard, active, pointing, vertical } = props
+
+  if (iconOnly) {
+    return { color: colors.border.focused }
+  }
+
   if (active && !underlined) return {}
+
+  if (underlined && !isFromKeyboard) {
+    return { color: v.focusedColor }
```

Comment:
It's a little confusing because these variables are mixed with color scheme variables throughout the file. What is the right paradigm to follow? Do we use color scheme for everything, except with a few minor, specific exceptions or "overrides"? It looks like this might be the only time this variable is used. If so and if we're only using non-color scheme variables for specific instances, can we rename this variable to something specific like `v.underlineInitialFocusedColor` or something?

Reply:
we need this to be a variable because this specific color should not change with the `color` prop (it's the same even if color is `red` or `green`)

regarding the name, it's debatable if we want to include something so specific... we need to come up with a naming convention for variables; if we take into account @codepretty 's proposal (`underlineInitialFocusedColor`) we can find the pattern to be:

`${propNames}${state}${area}Color`

where:
- `propNames`: is a concatenation of props that are set to true for the branch the color is used;
- `state`: is the state of the component (initial, focused, hovered, active, disabled, etc)
- `area`: is the area of the component that gets the color (border, background, foreground, etc)